pkg/kubectl/describe/versioned/describe.go

```diff
@@ -670,6 +670,8 @@ func describePod(pod *corev1.Pod, events *corev1.EventList) (string, error) {
 		w.Write(LEVEL_0, "Namespace:\t%s\n", pod.Namespace)
 		if pod.Spec.Priority != nil {
 			w.Write(LEVEL_0, "Priority:\t%d\n", *pod.Spec.Priority)
+		}
+		if len(pod.Spec.PriorityClassName) > 0 {
 			w.Write(LEVEL_0, "PriorityClassName:\t%s\n", stringOrNone(pod.Spec.PriorityClassName))
```

Comment:
prefer "Priority Class Name"

Reply:
The original one is `PriorityClassName`, I also think `Priority Class Name` is better. Done!